beacon-chain/blockchain/service.go

```diff
@@ -44,15 +48,17 @@ func DefaultConfig() *Config {
 func NewChainService(ctx context.Context, cfg *Config, beaconChain *BeaconChain, beaconDB *database.DB, web3Service *powchain.Web3Service) (*ChainService, error) {
 	ctx, cancel := context.WithCancel(ctx)
 	return &ChainService{
-		ctx:                              ctx,
-		chain:                            beaconChain,
-		cancel:                           cancel,
-		beaconDB:                         beaconDB,
-		web3Service:                      web3Service,
-		latestBeaconBlock:                make(chan *types.Block, cfg.BeaconBlockBuf),
-		processedBlockHashes:             [][32]byte{},
-		processedActiveStateHashes:       [][32]byte{},
-		processedCrystallizedStateHashes: [][32]byte{},
+		ctx:                                    ctx,
+		chain:                                  beaconChain,
+		cancel:                                 cancel,
+		beaconDB:                               beaconDB,
+		web3Service:                            web3Service,
+		latestBeaconBlock:                      make(chan *types.Block, cfg.BeaconBlockBuf),
+		canonicalBlockAnnouncement:             make(chan *types.Block, cfg.AnnouncementBuf),
```

Comment:
Is there a smaller word you can use other than Announcement? 

In go, we should prefer the smallest variable names possible without losing context or information. Maybe `Event` would be a good substitute?